platforms/ios/src/TangramMap/TGMapViewController.mm

```diff
@@ -135,7 +136,13 @@ - (void)loadSceneFile:(NSString *)path sceneUpdates:(NSArray<TGSceneUpdate *> *)
     }
 
     self.scenePath = path;
-    self.map->loadScene([path UTF8String], false, updates);
+
+    auto updateCallbackStatus = [self](auto sceneUpdateStatuses) {
+        if (!self.mapViewDelegate || ![self.mapViewDelegate respondsToSelector:@selector(mapView:didCompleteSceneUpdatesWithErrors:)]) { return; }
+        [self.mapViewDelegate mapView:self didCompleteSceneUpdatesWithErrors:[TGSceneUpdateStatus sceneUpdateStatuses:sceneUpdateStatuses]];
```

Comment:
Hmmm now that I see this in usage, would this be better named `didCompleteSceneUpdatesWithStatus` or is this only going to return when there are errors? Just getting tripped up on the name of things since typical Apple objc syntax is to return something named an error when you say `withErrors` - not a status.